### ISSUE TITLE
[Bug]: Model attributes are not autocompleted when using Sushi `$schema` property

### ISSUE THREAD

**Issue by aguingand** [laravel-idea/plugin#737]
### Bug description

According to https://github.com/laravel-idea/plugin/issues/171, Laravel Idea handle Models using Sushi but it does not seems to check the `$schema` property during Helper Code generation. (cf [Sushi docs](https://github.com/calebporzio/sushi#custom-schema))

However the code generation works well when `$rows` is defined.

### Plugin version

6.3.1.223

### Operating system

MacOS

### Steps to reproduce

- In a laravel project, install `calebporzio/sushi` package.
- Create a model using Sushi
```php
use Illuminate\Database\Eloquent\Model;
use Sushi\Sushi;

class Page extends Model
{
    use Sushi;

    protected array $schema = [
        'name' => 'string',
    ];

    public function getRows()
    {
        return []
    }
}
```
- Click "Generate Helper code"
- Go to the `vendor/_laravel_idea/_ide_helper_models_*.php` file containing the model definition
- Attributes defined in `$schema` are not present.

**Comment by adelf** [comment]
Fixed thanks. It used $rows as the main information about table fields.

**Comment by aguingand** [comment]
Thank you !

**Comment by adelf** [comment]
Fixed in 7.0
